src/components/data-management/MetadataEditor.jsx

```diff
@@ -1,26 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import {
-  Popover, Button, Space, Divider,
+  Popover, Button, Space, Divider, Tooltip,
 } from 'antd';
 import { FormatPainterOutlined } from '@ant-design/icons';
 
 const MetadataEditor = (props) => {
   const {
-    onCreate, onCancel, children, massEdit, ...restOfProps
+    onReplaceEmpty,
+    onReplaceAll,
+    onClearAll,
+    massEdit,
+    children,
+    ...restOfProps
   } = props;
 
+  const [value, setValue] = useState('');
+
+  const onChange = (e) => setValue(e?.target?.value || e);
+
   const getContent = () => (
     <Space direction='vertical'>
-      {children}
+      {React.cloneElement(children, {
+        onChange,
+      })}
       <Divider style={{ margin: '4px 0' }} />
 
       {massEdit
         ? (
           <Space>
-            <Button type='primary' size='small'>Fill all missing</Button>
-            <Button size='small'>Replace all</Button>
-            <Button type='warning' size='small'>Clear all</Button>
+            <Button
+              type='primary'
+              size='small'
+              onClick={() => {
+                onReplaceEmpty(value);
+              }}
+            >
+              Fill all missing
```

Comment:
The mockup has different labels, I think
![image](https://user-images.githubusercontent.com/460418/117250020-dcc67400-ae42-11eb-9084-97ac07d179cb.png)

Reply:
Yea, the label has been changed. There was a comment on the labels, but I think it's checked as done. I've also asked @vickymorrison to review the staging env.

the no-mass edit `Save` button is currently used only on the file columns. It would only appear if data is missing. I'm not sure about it's use yet though. Maybe @cosa65 can give some light?